plugins/modules/simpleinit_msb.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2016-2023, Vlad Glagolev <scm@vaygr.net>
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: simpleinit_msb
+short_description: Manage services on Source Mage GNU/Linux
+version_added: 7.5.0
+description:
+  - Controls services on remote hosts using C(simpleinit-msb).
+requirements:
+  - ansible-core 2.15.5 or newer
```

Comment:
This will be listed as "The below requirements are needed on the host that executes this module." ([example](https://docs.ansible.com/ansible/devel/collections/community/general/ufw_module.html#requirements)), which doesn't apply here. Better do something like
```
notes:
  - This module needs ansible-core 2.15.5 or newer. Older versions have a broken and insufficient daemonize functionality.
```